pkg/util/constants.go

```diff
@@ -58,6 +58,8 @@ const (
 	// resourcetemplate.karmada.io/retain-replicas: true   // with value `true` indicates retain
 	// resourcetemplate.karmada.io/retain-replicas: false  // with value `false` and others, indicates not retain
 	RetainReplicasLabel = "resourcetemplate.karmada.io/retain-replicas"
+
+	ResourceTemplateMatchedByLabel = "resorucetemplate.karmada.io/matched-by"
```

Comment:
How do you think the:
```suggestion
	ResourceTemplateMatchedByLabel = "resorucetemplate.karmada.io/claimed-by"
```